layouts/partials/navigation.html

```diff
@@ -1,8 +1,8 @@
 <nav class="navbar navbar-expand-md {{ if .IsHome }}navbar-dark{{ else }}navigation{{ end }}">
```

Comment:
This line seems to be causing problems with the navbar menu appearing or not. It will show on the homepage, but when you go to any other page, it is invisible on the background. I'm not sure why the theme is doing this `if` check here for the home page or any other page on the site.

This is worth digging into next week, so the navbar menu button will appear on all pages, not just the home page.

Reply:
@jwflory the homepage's header is different from that of other pages. The `if` check is to make sure of that.

Reply:
@Idadelveloper Right – my question is, why?